src/vendors/oasisscan/swagger3.yml

```diff
@@ -504,3 +548,25 @@ components:
         - epochLeft
         - validatorName
         - icon
+    CtxRow:
```

Comment:
This seems paratime-transaction specific; I would rename.

Reply:
This corresponds to `ctx` property of ParaTime transaction detail API. Should I rename?

Reply:
done